fastlane/CommonFastfile

```diff
@@ -121,7 +121,11 @@ lane :github_changelog do |options|
   # Temporary workaround to get around GitHub rate limit
   rate_limit_sleep = ENV["GITHUB_RATE_LIMIT_SLEEP"].to_i
 
-  formatted = commits.map do |commit|
+  supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test"]
```

Comment:
It will be good to move this to the fastlane plugin once we start using it. 🎉 for shared code and tests!

Reply:
yes! that's the idea :)